src/scripts/components/story/story.tsx

```diff
@@ -78,8 +80,9 @@ const Story: FunctionComponent = () => {
         <Header
           backLink={`/${mode.toString()}`}
           backButtonId="backToStories"
-          title={storyListItem.title}
-        />
+          title={storyListItem.title}>
+          <Button icon={ShareIcon} onClick={() => console.log('placeholder')} />
```

Comment:
Because it's used multiple times we should make a ShareButton component out of it

Reply:
Yes makes sense, i will also create a separate PR for that after my vacations. :)